arch/arm/src/imx6/imx_enet.c

```diff
@@ -200,6 +201,16 @@
 #  define BOARD_PHY_10BASET(s)  (((s) & MII_DP83825I_PHYSTS_SPEED) != 0)
 #  define BOARD_PHY_100BASET(s) (((s) & MII_DP83825I_PHYSTS_SPEED) == 0)
 #  define BOARD_PHY_ISDUPLEX(s) (((s) & MII_DP83825I_PHYSTS_DUPLEX) != 0)
+#elif defined(CONFIG_ETH0_PHY_AR8031)
+#  define BOARD_PHY_NAME        "AR8031"
+#  define BOARD_PHYID1          MII_PHYID1_AR8031
+#  define BOARD_PHYID2          MII_PHYID2_AR8031
+#  define BOARD_PHY_STATUS      MII_AR8031_PSSR
+#  define BOARD_PHY_ADDR        (1)
+#  define BOARD_PHY_10BASET(s)   (((s)&MII_AR8031_PSSR_10MBPS) == ((s)&MII_AR8031_PSSR_SPEEDMASK))
+#  define BOARD_PHY_100BASET(s)  (((s)&MII_AR8031_PSSR_100MBPS) == ((s)&MII_AR8031_PSSR_SPEEDMASK))
+#  define BOARD_PHY_1000BASET(s) (((s)&MII_AR8031_PSSR_1000MBPS) == ((s)&MII_AR8031_PSSR_SPEEDMASK))
+#  define BOARD_PHY_ISDUPLEX(s)  (((s)&MII_AR8031_PSSR_DUPLEX) != 0)
```

Comment:
Optional: would be good to add 1 space for all above PHY defines. The other option is `BOARD_PHY_1000BASET` -> `BOARD_PHY_1MBASET`
```suggestion
#  define BOARD_PHY_10BASET(s)   (((s) & MII_AR8031_PSSR_10MBPS) == ((s)&MII_AR8031_PSSR_SPEEDMASK))
#  define BOARD_PHY_100BASET(s)  (((s) & MII_AR8031_PSSR_100MBPS) == ((s)&MII_AR8031_PSSR_SPEEDMASK))
#  define BOARD_PHY_1000BASET(s) (((s) & MII_AR8031_PSSR_1000MBPS) == ((s)&MII_AR8031_PSSR_SPEEDMASK))
#  define BOARD_PHY_ISDUPLEX(s)  (((s) & MII_AR8031_PSSR_DUPLEX) != 0)
```
if suggestion does not exceed line size limitation.

Reply:
Done.